src/DataOneV2/DataOneV2.jl

```diff
@@ -4,30 +4,36 @@ export KNB, TERN
 include("KNB.jl")
 include("TERN.jl")
 
-description(repo::DataOneV2, mainpage) = desc_(repo, mainpage)
+description(repo::DataOneV2, mainpage) = try desc_(repo, mainpage) catch missing end
 
-author(repo::DataOneV2, mainpage) = authors_(repo, mainpage)
+author(repo::DataOneV2, mainpage) = try authors_(repo, mainpage) catch missing end
 
-license(repo::DataOneV2, mainpage) = license_(repo, mainpage)
+license(repo::DataOneV2, mainpage) = try license_(repo, mainpage) catch missing end
 
-published_date(repo::DataOneV2, mainpage) = pub_date(repo, mainpage)
-
-get_urls(repo::DataOneV2, page) = []
+published_date(repo::DataOneV2, mainpage) = try pub_date(repo, mainpage) catch missing end
 
 function license_(repo::DataOneV2, mainpage)
```

Comment:
rename this to 
`function license(repo::DataOneV2, mainpage)`

and delete the definition on line 11.

The others also need the same done, but that can come later.
This one is particularly bothersome,
as it is double wrapped in `try`